ffi/build.rs

```diff
@@ -26,11 +26,9 @@ fn main() {
         .header("src/shim.h")
         .generate_comments(true)
         .generate_inline_functions(true)
-        .whitelist_type(r"(rte|cmdline|ether|eth|arp|vlan|vxlan)_.*")
-        .whitelist_function(r"(_rte|rte|cmdline|lcore|ether|eth|arp|is)_.*")
-        .whitelist_var(
-            r"(RTE|CMDLINE|ETHER|ARP|VXLAN|BONDING|LCORE|MEMPOOL|ARP|PKT|EXT_ATTACHED|IND_ATTACHED|lcore|rte|cmdline|per_lcore)_.*",
-        )
+        .whitelist_type(r"(rte|eth|ether)_.*")
+        .whitelist_function(r"(_rte|rte|eth|ether|lcore)_.*")
```

Comment:
as mentioned in chat, we may not need these lcore/per_lcore whitelists, but maybe we can clean that up later once we're sure.

Reply:
yeah once all the calls to them are gone, we can remove them.